src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+:root {
+  --primary-color: #00acdc;
+}
+
+header {
+  justify-content: space-between;
+  display: flex;
+  background: #fff;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.ul {
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. Instead of `.ul`, use a class that describes the element's purpose, like `.nav__list`.